content/blog/2021/10/new-architecture-to-ease-kubewarden-administrators-lives.md

```diff
@@ -0,0 +1,257 @@
+---
+title: A new architecture to ease Kubewarden administrators' lives
+authors:
+    - Víctor Cuadrado Juan
+date: 2021-10-01
+---
+
+We are pleased to announce a new architecture for the Kubewarden stack, in line
+with its journey to maturity:
+
+The introduction of a **PolicyServer** Custom Resource Definition which allows
+users to describe a policy-server Deployment, together with binding
+**ClusterAdmissionPolicies** to a specific **PolicyServer** instance.
+
+These 2 changes are accompanied by a multitude of improvements to make Kubewarden
+more comfortable for Kubernetes Administrators, such as validation for
+Kuberwarden Custom Resources, improvements in Helm Charts, and Status and
+Conditions for **ClusterAdmissionPolicies**.
+
+For a comprehensive list, see the [release notes](TODO) of our latest Helm
+charts.
+
+## How does it look like?
+
+In previous versions, the Kubewarden Controller instantiated a single Deployment
+of policy-server. That policy-server was configured via a ConfigMap, which
+contained the Deployment options (image, replicas, …), and a list of policies to
```

Comment:
```suggestion
contained the Deployment options (image, replicas, ...), and a list of policies to
```

I always find it weird when using the unicode endpoint for ellipsis.

Reply:
Fixed in https://github.com/kubewarden/kubewarden.io/pull/33/commits/c2429ef94810f7aaa7cee8ed97120415ef090969.